src/pkcs11/pkcs11-spy.c

```diff
@@ -400,6 +400,116 @@ spy_attribute_list_out(const char *name, CK_ATTRIBUTE_PTR pTemplate,
 	print_attribute_list(spy_output, pTemplate, ulCount);
 }
 
+static void
+spy_dump_mechanism_in(const char *name, CK_MECHANISM_PTR pMechanism)
```

Comment:
now, the first argument "name" is bogus as it is always "pMechanism" so we can probably drop it too for simplicity.

Reply:
That would change the convention vs. above methods -- can do if you want.

Reply:
oh, right. Lets keep it as it is. Thanks for checking!